docs/en/compute/HBase.md

```diff
@@ -30,29 +30,42 @@ Therefore, the configuration of Alluxio is done mostly in HBase configuration fi
 
 ### Set property in `hbase-site.xml`
 
-Change the `hbase.rootdir` property in `conf/hbase-site.xml`:
-> You do not need to create the `/hbase` directory in Alluxio, HBase will do this for you.
+Set the following properties in `conf/hbase-site.xml` and make sure all HBase cluster nodes
+have the configuration.
 
+Set the `hbase.rootdir` property as follows: 
 ```xml
 <property>
   <name>hbase.rootdir</name>
   <value>alluxio://master_hostname:port/hbase</value>
 </property>
 ```
 
-Add the following property to the same file `hbase-site.xml`.
-(make sure it is configured in all HBase cluster nodes):
+> You do not need to create the `/hbase` directory in Alluxio, HBase will do this for you.
 
+Add the following property to the same file `hbase-site.xml`:
 ```xml
 <property>
   <name>hbase.regionserver.hlog.syncer.count</name>
   <value>1</value>
 </property>
 ```
 
-This property is required to prevent HBase from flushing Alluxio file stream in a thread unsafe
+> This property is required to prevent HBase from flushing Alluxio file stream in a thread unsafe
 way.
 
+If you are running HBase version greater than 2.0, add the following property:
+
+```xml
+<property>
+  <name>hbase.unsafe.stream.capability.enforce</name>
+  <value>false</value>
+</property>
+```
+
+> This will disable HBase new stream capabilities (hflush/hsync) used for WAL.
+HBASE-14790, HBASE-15536
```

Comment:
This has to be configured for HBase >2.0. The related changes are HBASE-14790 and HBASE-15536. Should I leave a ref to those changes?